src/Illuminate/Foundation/Console/BroadcastingInstallCommand.php

```diff
@@ -168,9 +168,17 @@ protected function installNodeDependencies()
             ];
         }
 
-        Process::command(implode(' && ', $commands))
-            ->path(base_path())
-            ->tty(true)
-            ->run();
+        $command = Process::command(implode(' && ', $commands))
+                        ->path(base_path());
+
+        if (strtoupper(substr(PHP_OS, 0, 3)) !== 'WIN') {
```

Comment:
```suggestion
        if (! windows_os()) {
```

`windows_os` function available from `Illuminate\Support\helpers.php`.